plugins/search-backend-node/src/index.ts

```diff
@@ -22,6 +22,7 @@
 
 export { IndexBuilder } from './IndexBuilder';
 export { Scheduler } from './Scheduler';
+export * from './collators';
```

Comment:
I read [this ADR](https://backstage.io/docs/architecture-decisions/adrs-adr004) to mean that higher-level `index.ts` files should almost exclusively use `*` exports, and leaf-level `index.ts` files should be used to control what is publicly exported or not.

As we move toward a 1.0 version of search, we should probably clean up exports that don't adhere to this pattern.  ...Although, since all exports are tracked in `api-report.md` files...  It almost doesn't matter anymore.